Index: recipe/gmpy2-2.0.8/src/gmpy_mpfr.c
===================================================================
--- gmpy2-2.0.8.orig/src/gmpy_mpfr.c
+++ gmpy2-2.0.8/src/gmpy_mpfr.c
@@ -753,7 +753,7 @@ _mpfr_hash(mpfr_t f)
             else
                 return -_PyHASH_INF;
         else
-            return _PyHASH_NAN;
+            return 0;
     }
 
     /* Calculate the number of limbs in the mantissa. */
